pysatMadrigal/instruments/madrigal_pandas.py

```diff
@@ -75,7 +75,9 @@
 name = 'pandas'
 tags = dict()
 pandas_format = True
-excluded_tags = ['8105']  # Pandas-style data that requires special support
+
+# Pandas-style data that requires special support
+excluded_tags = ['120', '180', '210', '211', '212', '8105']
```

Comment:
May want to consider some form of message to the user if they try to load an excluded tag. In the code changes highlighting I don't see where this excluded_tag gets used.

Reply:
Look a few lines down (87).  They get the standard python "unknown tag" warning, like any other instrument.